### PR TITLE
Admin control da

### PR DESCRIPTION
**Pull request by atkinson-daniel**
Update flash messages to either: :success, :error, :message, or :inventory_notice and added new CSS styling for the last two

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -140,6 +140,23 @@ form {
...
+  padding: 10px;
+  background-color: #4b98e4;
+  color: white;
+}
+
+.inventory_notice-flash {
+  padding: 10px;
+  background-color: #5134c3;
+  color: white;
+}
```
Comment (StephanieFriend) [inline app/assets/stylesheets/application.css:159]: NICE!

File: app/controllers/cart_controller.rb [hunk 2]
```diff
@@ -29,7 +29,7 @@ def increment
         if quantity < item.inventory
           cart.contents[item_id] += 1
         else
-          flash[:inventory_notice] = "Do not have enough of this item in stock, please choose another."
+          flash[:inventory_notice] = "There's not enough of this item in stock, please choose another."
         end
```
Comment (StephanieFriend) [inline app/controllers/cart_controller.rb:33]: I like this much better

File: spec/features/cart/cart_qantity_spec.rb [hunk 3]
```diff
@@ -30,7 +30,7 @@
           expect(page).to have_content(2)
         end
 
-        expect(page).to have_content("Do not have enough of this item in stock, please choose another.")
+        expect(page).to have_content("here's not enough of this item in stock, please choose another.")
       end
```
Comment (StephanieFriend) [inline spec/features/cart/cart_qantity_spec.rb:34]: Forgot the T!

**Review by StephanieFriend — CHANGES_REQUESTED** [review]
This looks great! Only thing is a spelling error where you forgot a T! It's in cart_quantity_spec

File: spec/features/cart/cart_qantity_spec.rb (hunk 3 above)
Comment (atkinson-daniel) [reply spec/features/cart/cart_qantity_spec.rb:34]: Fixed!

**Pull request merged**